lib/ansible/modules/utilities/logic/async_wrapper.py

```diff
@@ -240,9 +240,10 @@ def _run_module(wrapped_cmd, jid, job_path):
                 "failed": 1,
                 "msg": "could not create: %s" % jobdir
             }))
+            sys.exit(1)
```

Comment:
i would just substitute the print in this case

Reply:
But the json payload is needed for debugging purposes, no?

A sole return code of 1 not very descriptive.

Reply:
i meant
```            sys.exit(json.dumps({
￼	                "failed": 1,
￼	                "msg": "could not create: %s" % jobdir
￼	            }))
```

but nvmd since that will go to stderr and we want it in stdout anyways, so you can ignore the comment.